AmplifyPlugins/Core/AWSPluginsCore/Model/Decorator/ModelIdDecorator.swift

```diff
@@ -33,7 +33,7 @@ public struct ModelIdDecorator: ModelBasedGraphQLDocumentDecorator {
     public init(identifierFields: [(name: String, value: Persistable)]) {
         var firstField = true
         identifierFields.forEach { name, value in
-            self.identifierFields.append((name: name, value: "\(value)", type: firstField == true ? "ID!" : "String!"))
+            self.identifierFields.append((name: name, value: Self.convert(persistable: value), type: firstField == true ? "ID!" : "String!"))
```

Comment:
This PR modifies the GraphQL `get` queries by substituting inline values instead of variables. It disregards the `type` information and directly inserts the value into the query. The intention is to preserve the existing logic of the `type` since other queries also rely on the `ModelIdDecorator` for GraphQL query generation, and avoiding any potential disruptions.